inst/stan/wwinference.stan

```diff
@@ -190,15 +227,49 @@ transformed parameters {
   // Site level disease dynamic estimates!
   i0_site_over_n = inv_logit(logit(i0_over_n) + eta_i0 * sigma_i0);
   growth_site = initial_growth + eta_growth * sigma_growth; // site level growth rate
+
+  // Site level spatial Rt------------------------------------------------------
+  if (ind_corr_func == 1){
```

Comment:
If you use an `int<lower=0, upper=1>`, can just call if
```suggestion
  if (ind_corr_func){
```

Reply:
See also https://mc-stan.org/docs/reference-manual/removals.html#real-values-in-conditionals